tests/ui-toml/toml_import_rename/conf_import_rename.rs

```diff
@@ -0,0 +1,13 @@
+#![warn(clippy::import_rename)]
+
+extern crate regex;
+extern crate serde;
+extern crate syn;
+
+use regex::bytes as xegersetyb;
+use serde::{de::Deserializer as WrongRename, ser::Serializer as DarkTower};
+use syn::Path as SynPath;
```

Comment:
Missing a test without `as rename`.

Reply:
Also need a test without `as rename` that is linted.

Reply:
There that should do it, my bad.